src/index.js

```diff
@@ -1,6 +1,16 @@
-'use strict';
-export DistinguishedName from './dn';
-export RelativeDistinguishedName from './rdn';
-export escape from './escape';
-export format from './format';
-export parse from './parse';
+import DistinguishedName from './dn';
+import RelativeDistinguishedName from './rdn';
+import escape from './escape';
+import format from './format';
+import parse from './parse';
+
+export { DistinguishedName, RelativeDistinguishedName, escape, format, parse };
```

Comment:
FWIW you might want to check that this is still providing the behavior you intended. I was having some trouble getting the newer version of Babel to generate things like they were before.